frontend/src/app/pages/exercises/exercise/shared/editor-panel/edit-vehicle-template-modal/edit-vehicle-template-modal.component.ts

```diff
@@ -41,6 +48,20 @@ export class EditVehicleTemplateModalComponent implements OnInit {
                 this.store
             )
         );
+        this.materialTemplates = cloneDeepMutable(
+            Object.values(
+                selectStateSnapshot(selectMaterialTemplates, this.store)
+            ).filter((template) =>
+                this.vehicleTemplate!.materialTemplateIds.includes(template.id)
+            )
+        );
+        this.personnelTemplates = cloneDeepMutable(
+            Object.values(
+                selectStateSnapshot(selectPersonnelTemplates, this.store)
+            ).filter((template) =>
+                this.vehicleTemplate!.personnelTemplateIds.includes(template.id)
+            )
+        );
```

Comment:
This does not take the count of the personnel/material into account. If I create a vehicle with three instances of big material, save the template and open the editor again, it only shows one instance. Immediately saving the template again (without performing any changes, which should be a no-op) then actually writes the wrong number of big material into the template so that subsequently added vehicles only have a single big material.

Same issue for the personnel.

You can easily reproduce the issue by editing the "GW San" template that consists of 2 RS, 2 San, 4 big material (and one GF and NA each). The editor does not show those counts.